client/src/components/AddChild/addchild.js

```diff
@@ -1,19 +1,19 @@
 import React, { Component } from "react";
-import DeleteBtn from "../DeleteBtn";
 import Jumbotron from "../Jumbotron";
 import API from "../../utils/API";
-import { Link } from "react-router-dom";
-import { Col, Row, Container } from "../Grid";
-import { List, ListItem } from "../List";
-import { Input, TextArea, FormBtn } from "../Form";
+import { List } from "../List";
+import { Input, FormBtn } from "../Form";
 import { Redirect } from "react-router-dom";
+import { Modal,Card,Col } from 'react-materialize';
+import Button from 'react-bootstrap/Button';
+
 
 
 class AddChild extends Component {
```

Comment:
a class named as add child seems like a bad name for a component. Does this whole component add a child? This to me would be a function name that will add a child. To me this would better be called like AddChildModal or something like that. or addChildForm. Just add child isn't a good class name.

Reply:
we will review on next round of changes the naming schema